src/schema.ts

```diff
@@ -110,32 +110,32 @@ const typeDefs = `#graphql
     """
     Updates an address by id
     """
-    updateAddress(_id: ID!, zipCode: Int, street: String, houseNumber: Int): Address
+    updateAddress(_id: ID!, zipCode: Int, street: String, houseNumber: Int, token: String!): Address
     """
     Deletes an address by id
     """
-    deleteAddress(_id: ID!): Address
+    deleteAddress(_id: ID!, token: String!): Address
     """
     Create Admin for company
     """
-    createAdmin(firstName: String!, lastName: String!, email: String!, password: String!, role: Role!, zipCode: Int!, street: String!, houseNumber: Int!, companyId: ID!): User
+    createAdmin(firstName: String!, lastName: String!, email: String!, password: String!, role: Role!, zipCode: Int!, street: String!, houseNumber: Int!, companyId: ID!, token: String!): User
```

Comment:
Could we rename this to createCompanyAdmin to make it clear that it is not a typical admin user